### PR TITLE
Write FASTA to file

### PR DESCRIPTION
**Pull request by wildgarden**
This PR allows to write FASTA sequences to files. The sequence is written in chunks that obey the maximum character per line restriction. I'm not quite happy how I iterate over the sequence to split it into chunks. Since I'm new to Julia I didn't find a more elegant approach that uses string manipulation/list-comprehension.

### CODE REVIEW

**Comment by dcjones** [comment]
LGTM :+1:

File: test/seq/test_seq.jl [hunk 1]
```diff
@@ -1292,4 +1292,37 @@ facts("Sequence Parsing") do
...
+        sequences = [fasta_seq1, fasta_seq2]
+
+        tempfile = tempname()
+        try
+            open(tempfile, "w") do fh
+                for seq in sequences
+                    write(fh, seq)
+                end
+            end
+            @fact readall(open(tempfile)) --> expected
```
Comment (bicycle1885) [inline test/seq/test_seq.jl]: `open(readall, tempfile)` can automatically close the temporary file after reading its whole content.

**Comment by wildgarden** [comment]
@bicycle1885 Thank you, I somehow overlooked this option and adopted the PR in  0f1170623e30ef457f2536867bd9edb524c99e9d 

To conform with the logic used in fastaq.jl the conversion of nt was changed in 4e31f99017c54364c4b82966ae8c533ac0c9eb22 to delegate it to function show().

**Comment by Ward9250** [comment]
Indeed, looks good.

**Comment by prcastro** [comment]
@wildgarden , this looks good. Could we rebase it on the latest `master` so it gets merged?

**Comment by dcjones** [comment]
I've rebased and merged this to save @wildgarden the headache.

**Comment by Ward9250** [comment]
@dcjones Thanks :+1:

**Comment by wildgarden** [comment]
@dcjones Thank you!

**Pull request closed without merging**